src/components/Navbar/Navbar.tsx

```diff
@@ -27,6 +27,9 @@ function Navbar() {
             <li className="navbar-item">
               <Link to="/leaderboard"> Leaderboard </Link>
             </li>
+            <li className="navbar-item">
```

Comment:
suggestion: li is repeating with the same styling. So, it can be abstracted into a component.